staging/src/k8s.io/sample-controller/main.go

```diff
@@ -38,6 +38,7 @@ var (
 )
 
 func main() {
+	klog.InitFlags(nil)
```

Comment:
Note that where there are any remnants of glog floating around, we'll need to do something like https://github.com/kubernetes-sigs/cluster-api/pull/610

@dims do you think we should put something like that into klog?

Reply:
yep, opened an issue so i won't forget https://github.com/kubernetes/klog/issues/26